cmd/immuclient/audit/utils.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2019-2020 vChain, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package audit
+
+import (
+	"fmt"
+	"os"
+)
+
+// QuitToStdErr prints an error on stderr and closes
+func QuitToStdErr(msg interface{}) {
```

Comment:
We should use helper in https://github.com/codenotary/immudb/blob/master/cmd/helper/cmd.go#L102 to avoid duplicates. This change request is postponed after released